perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -200,14 +207,19 @@ def _ParseNetperfOutput(stdout, metadata, benchmark_name,
   # 99th Percentile Latency Microseconds,Minimum Latency Microseconds,
   # Maximum Latency Microseconds\n
   # 1405.50,Trans/s,2.522,4,783.80,683,735,841,600,900\n
-  fp = io.StringIO(stdout)
-  # "-o" flag above specifies CSV output, but there is one extra header line:
-  banner = next(fp)
-  assert banner.startswith('MIGRATED'), stdout
-  r = csv.DictReader(fp)
-  results = next(r)
-  logging.info('Netperf Results: %s', results)
-  assert 'Throughput' in results
+  try:
+    fp = io.StringIO(stdout)
+    # "-o" flag above specifies CSV output, but there is one extra header line:
+    banner = next(fp)
+    assert banner.startswith('MIGRATED'), stdout
+    r = csv.DictReader(fp)
+    results = next(r)
+    logging.info('Netperf Results: %s', results)
+    assert 'Throughput' in results
+  except:
```

Comment:
It's for robustness/debugging - if some of the threads fail, we'll output the stdout that caused them to fail and just calculate aggregate stats on the threads that succeeded. We report how many threads succeeded. If all the threads fail, we raise an exception.